docs/content/https/ref-acme.toml

```diff
@@ -30,6 +30,13 @@
   #
   # certificatesDuration=2160
 
+  # Timeout for HTTP responses from the ACME server. 
+  #
+  # Optional
+  # Default: 30s
+  #
+  # clientTimeout=30s
+
```

Comment:
The `clientResponseHeaderTimeout` option is missing here and there.

Reply:
Somehow I though that `make generate` would update those files too.
Did that manually in   https://github.com/traefik/traefik/pull/11637/commits/c5b6bd3f7142a4767b6cc2f868856c70b4d0c683.